web/templates/user_group_info_popover.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="popover message-info-popover group-info-popover">
+<div class="popover message-user-card-popover group-info-popover">
```

Comment:
This probably shouldn't have the `message-info-popover` class at all -- but I guess that can be cleaned up when working on that popover itself.

Reply:
Yeah, it was resolved in #26742.